index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title></title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="style.css" rel="stylesheet">
+</head>
+
+<body>
+    <div id="main-page">
+        <div>
+            <section id="navbar">
+                <header id="header">Js Documentation</header>
+
+
+                <a href="#introduction" class="active">
+                    <header class="nav-link">Introduction</header>
+                </a>
+
+
+                <a href="#what_you_should_already_know" class="active">
+                    <header class="nav-link">what you should already know</header>
+                </a>
+
+
+                <a href="#javaScript_and_Java" class="active">
+                    <header class="nav-link">JavaScript and Java</header>
+                </a>
+
+
+                <a href="#hello_world" class="active">
+                    <header class="nav-link">Hello world</header>
+                </a>
+
+
+                <a href="#variables" class="active">
+                    <header class="nav-link">Variables</header>
+                </a>
+
+
+
```

Comment:
index.html file has a lot of new empty lines. to make the code neater as well as the page run faster all unnecessary lines and spaces should be fixed.